cmd/istioctl/main.go

```diff
@@ -99,6 +99,7 @@ var (
 			if len(args) != 0 {
 				return fmt.Errorf("create takes no arguments")
 			}
+			c.SilenceUsage = true
```

Comment:
SilenceUsage is not an _istioctl_ flag.  It is an internal option to Cobra that effects how Cobra prints error messages.  However at Kuat's request I have changed the logic and the new logic is much more clear.